spec/acceptance/class_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper_acceptance'
 
 describe 'proxysql class' do
-  unless fact('os.release.major') == '18.04' ||
+  unless fact('os.release.major') >= '18.04' ||
```

Comment:
I don't think we can use versioncmp for the tests, and since that's what we use in the code that would probably not make much sense…

Maybe we can list the versions where we want/do not want to do something?

```suggestion
  unless ['18.04', '20.04'].include?(fact('os.release.major')) ||
```

This is likely to fail when someone wants to add support for a future release but it should be an easy fix.


Same for the other comparisons in the test suite.